### PR TITLE
Dashboard UX improvements

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Fix dropdown focus issues
  - Keyboard navigation to/from dropdowns and between options
  - Mouse focus to/from dropdowns (there is no mouse navigation between options)
- Use new inputs in "new secret" modal
- Simplify form types
- Switch activity log date filters to use `DatePicker` which is a wrapper around a `react-aria-components` input
  - Update `react-aria-components`
- Prevent Navigator2D from handling event if it has already been `defaultPrevented`
- Switch from `*` imports to `{}` imports for various files
- Switch assets table row headings to use new components

### Important Notes
None

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] ~~Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.~~
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] ~~Unit tests have been written where possible.~~

### CODE REVIEW

**Review by vitvakatu — APPROVED** [review]
GUI files

File: app/ide-desktop/common/src/utilities/data/object.ts [hunk 1]
```diff
@@ -66,6 +66,19 @@ export function unsafeEntries<T extends object>(
   return Object.entries(object)
 }
 
+// =====================
+// === unsafeEntries ===
```
Comment (kazcw) [inline app/ide-desktop/common/src/utilities/data/object.ts]: Wrong header

**Review by kazcw — APPROVED** [review]
Reviewed only common code

**Comment by PabloBuchu** [comment]
QA ✅

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**